hnn_core/tests/test_gui.py

```diff
@@ -50,6 +54,36 @@ def test_gui_load_params():
     plt.close('all')
 
 
+def test_prepare_upload_file():
+    """Tests that input files from local or url sources import correctly"""
+    def _import_json(content):
+        decode = codecs.decode(content, encoding="utf-8")
+        json_content = json.load(io.StringIO(decode))
+        return json_content
+
+    url = "https://raw.githubusercontent.com/jonescompneurolab/hnn-core/master/hnn_core/param/default.json"  # noqa
+    file = Path(hnn_core_root, 'param', 'default.json')
+
+    content_from_url = _prepare_upload_file(url)[0]
+    content_from_local = _prepare_upload_file(file)[0]
+
+    assert (content_from_url['name'] ==
+            content_from_local['name'] ==
+            'default.json')
+    assert (content_from_url['type'] ==
+            content_from_local['type'] ==
+            'application/json')
+    # Check that the size attribute is present. Cannot do an equivalency check
+    # because file systems may add additional when saving to disk.
+    assert content_from_url.get('size')
+    assert content_from_local.get('size')
```

Comment:
```suggestion
    assert 'size' in content_from_url
    assert 'size' in content_from_local
```
Little more readable